homu/main.py

```diff
@@ -286,6 +289,25 @@ def parse_commands(body, username, repo_cfg, state, my_username, db, *, realtime
 
             state.save()
 
+        elif word.startswith('delegate='):
+            if try_only:
+                state.add_comment(':key: Insufficient privileges')
+                continue
+            state.delegate = word[len('delegate='):]
+            state.save()
+
+        elif word == 'delegate-':
+            state.delegate = ''
+            state.save()
+
+        elif word == 'delegate+':
+            if try_only:
+                state.add_comment(':key: Insufficient privileges')
+                continue
+            state.delegate = state.get_repo().pull_request(state.num).user
```

Comment:
`pull_request(...).user` returns a `User`, but it is compared to a plain string in the authentication code above (`if username == state.delegate`). It would be better to store `pull_request(...).user.login` instead here, to keep the logic simple.